rocketpy/AeroSurfaces.py

```diff
@@ -633,12 +633,25 @@ def rollInfo(self):
                 self.rollForcingInterferenceFactor
             )
         )
-
+        # lacks a title for the plot
         self.rollParameters[0]()
+        # lacks a title for the plot
         self.rollParameters[1]()
 
         return None
 
+    def airfoilPlots(self):
+
+        if self.airfoil is not None:
```

Comment:
I think it would be better to create a method called `airfoilInfo` and there add this plot to it

Also inside `airfoilInfo` I think there should two prints at the beginning:
```
print("\n\nAerodynamic Information")
print()
print("Airfoil's Lift Curve as a Function of Alpha ({})).format(self.airfoil[1]))
```

Reply:
Maybe:
```
print("\n\nAerodynamic Information\n")
```